src/components/Organization/index.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react'
+import { bindActionCreators } from 'redux'
+import { connect, Dispatch } from 'react-redux'
+import {
+	Button,
+} from 'react-bootstrap'
+
+// Components
+import TableComponent from '../Shared/Table'
+
+// State
+import { IRootReducerState } from '../../reducers'
+import { IOrganizationState, IOrganization } from '../../reducers/organization'
+
+// Actions
+import { OrganizationActions, OrganizationDispatch } from '../../actions/organization'
+
+// State added to props after connect.
+interface IConnectedState {
+	organizationState: IOrganizationState,
+}
+
+// Actions added to props after connect.
+interface IConnectedActions {
+	organizationActions: OrganizationDispatch,
+}
+
+interface IComponentState {
+	confirmation: string
+}
+
+type Props = IOrganization
+type State = IOrganizationState
+
+class OrganizationViewComponent extends React.Component<any, State> {
```

Comment:
Your declaring that the `state` for the component with conform to the `State` interface.  However, you are setting an `organizations` property on `state`. While the compiler might not be complaining, you are still missing the `organizations` property in state interface declaration. 

Also, I believe you want to substitute the `any` for `Props`